### DISCUSSION TITLE
Connection Pooling

### DISCUSSION THREAD

**Discussion by vittolewerissa** [prisma/prisma#6473] [Q&A]
Hai guys I have a question 🙏

I am using PM2 where I can use it to cluster multiple instances, the question is, will each instance has its own query engine process or there's only 1 query engine and shared among instances? Are the connection pool limit is per instance or per query engine?

Thank you.

**Comment by vittolewerissa** [comment]
I think I got the answer from the doc.

> Multiple instances of PrismaClient results in multiple instances of the query engine. Each query engine creates a connection pool with a default pool size of num_physical_cpus * 2 + 1

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
